src/siddhant/Whileloop.java

```diff
@@ -0,0 +1,53 @@
+package siddhant;
+
+public class Whileloop{
+	int start;
+	int end;
+	void Evennumbers(int a){
+		end = a;
```

Comment:
you don’t require to create extra variable, you can handle this without it, think on this

please refer below code
https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/208/files